crates/anthropic/src/anthropic.rs

```diff
@@ -797,17 +782,38 @@ pub struct MessageDelta {
     pub stop_sequence: Option<String>,
 }
 
-#[derive(Error, Debug)]
```

Comment:
Only `ApiError` now has `Error`, because this one doesn't get converted to an `anyhow::Error` anymore.